src/main/java/dev/j3fftw/litexpansion/GlowEnchant.java

```diff
@@ -61,9 +60,11 @@ public boolean canEnchantItem(ItemStack item) {
             final ItemMeta itemMeta = item.getItemMeta();
             final Optional<String> id = SlimefunPlugin.getItemDataService().getItemData(itemMeta);
             if (id.isPresent()) {
-                return id.get().equals(Items.NANO_BLADE.getItemId());
+                return id.get().equals(Items.ADVANCED_CIRCUIT.getItemId())
+                        || id.get().equals(Items.NANO_BLADE.getItemId());
```

Comment:
isnt there a better way to do this @WalshyDev

Reply:
I was thinking that maybe i could put these in an array and then define glowable items at the top

Reply:
it will look ugly really quick

Reply:
Not really, at least, not a way that matters to do

Reply:
Probably should be in an ImmutableSet or something but... can't do that in Bukkit IIRC and we don't run J9.